pages/FAQ/_index.md

```diff
@@ -383,3 +383,19 @@ from Steam.
 
 Steam has its own file picker, however, it's not functional. Install
 `xdg-desktop-portal-gtk` to show the desktop's file picker.
+
+# Workspaces or clients are disappearing or monitor related dispatchers cause crashes
+
+It seems there is a Kernel bug making the system thinks there is an extra
+phantom monitor, that causes all sorts of issues, crashes and weird behaviors
+like disapearing workspaces or clients when adding or removing an external
+monitor.
+
+First check the list of monitors detected by Hyprland by running:
+
+`hyprctl monitors`
+
+If you see a monitor that should not be there (usually named `Unknown-1`), you
+can work around the issue by adding in your `hyprland.conf`:
+
+`monitor=Unknown-1,disabled`
```

Comment:
make it a full line code block with ini please
```ini
like=this
```

Reply:
done